src/routes/tag.js

```diff
@@ -0,0 +1,33 @@
+import express from 'express'
+import path from 'path'
+import groupBy from 'lodash/groupBy'
+import * as tag from '../database/queries/tag'
+const router = express.Router()
+
+router.get('/', function(req, res, next){
+  tag.getAll()
+  .then( results => {
+    const tags = groupBy(results, tag => tag.names)
+    res.json(tags)
+  })
+})
+
+router.post('/', function(req, res, next){
+  tag.add(req.body)
```

Comment:
this is an async function that returns a promise. You need to handle the resolution or rejection of this promise before you respond to the HTTP request.